ts/screens/onboarding/FingerprintScreen.tsx

```diff
@@ -40,19 +44,20 @@ export class FingerprintScreen extends React.PureComponent<Props, State> {
     };
   }
 
-  private renderBiometryType(biometryType: BiometrySimpleType): string {
-    switch (biometryType) {
-      case "FINGERPRINT":
-        return "Fingerprint";
-      case "FACE_ID":
-        return "FaceID";
-      case "TOUCH_ID":
-        return "TouchID";
-      case "NOT_ENROLLED":
-        return "Not Enrolled";
-      default:
-        return "Unavailable";
-    }
+  /**
+   * Print the only BiometrySimplePrintableType values that are passed to the UI
+   * @param biometrySimplePrintableType
+   */
+  private renderBiometryType(
+    biometryPrintableSimpleType: BiometryPrintableSimpleType
+  ): string {
+    return I18n.t(`onboarding.fingerprint.body.enrolledType.${
+      biometryPrintableSimpleType === "FINGERPRINT"
+        ? "fingerprint"
+        : biometryPrintableSimpleType === "FACE_ID"
+          ? "faceId"
+          : "touchId"
+    }` as TranslationKeys);
```

Comment:
if we change the translation keys, this cast will force the conversion and no error will be thrown by the compiler - instead use a switch statement like:

```ts
switch(biometryPrintableSimpleType) {
  case "FINGERPRINT":
    return I18n.t("onboarding.fingerprint.body.enrolledType.fingerprint");
    ...
}
```

Reply:
changed